joint_state_controller/test/test_common.hpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 PAL Robotics SL.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TEST_COMMON_HPP_
+#define TEST_COMMON_HPP_
+
+
+namespace joint_state_controller_testing
+{
+constexpr auto urdf =
```

Comment:
I would propose to use descriptions.hpp header for this as proposed in #99 and ros-controls/ros2_control#247.
We should just move the file in `test_robot_hardware` as @Karsten1987 proposed.

Reply:
https://github.com/ros-controls/ros2_controllers/issues/113 created follow-up issue